### ISSUE TITLE
When I try to start it Google Play opens and there is a toast about "Puzzels is missing a required file!..."

### ISSUE THREAD

**Issue by mobluse** [chrisboyle/sgtpuzzles#551]
**Describe the bug**
When I try to start it Google Play opens and there is a toast about "Puzzels is missing a required file! Try reinstalling from Google Play." If I don't move it to the external SD card, it works.

**To Reproduce**
1. I install the app using F-Droid
2. I move the app to the external SD card using settings Apps.
3. I start the app.
4. The error toast is shown and the app is stopped and it presentation is shown in Google Play.

**Expected behavior**
The app should start as if I hadn't moved it to the external SD card.

**Screenshots**

**Version info (optional, but sometimes helpful)**
 - Phone/tablet model: Alcatel 1C 2019 TCL 5003D_EEA - SDK: 27 ABI: armeabi-v7a
 - Android version: 8.1.0 Go
 - App version: 2022-04-30-1100-c43a34fb-fdroid

**Additional context**
A lot of people need to free up internal flash memory. If it cannot work on external SD card, it should not be possible to move it.

**Comment by yashpalgoyal1304** [comment]
> * Install the app using F-Droid (or Google Play)
> * Move the app to the external SD card using Settings/Apps/Puzzles/Storage.
> * Start the app.

hey, have u tried doing the "start the app" step-3 just after "install the app" step-1

**Comment by mobluse** [comment]
Yes, it works if I don't move it: "If I don't move it to the external SD card, it works."
It shouldn't be movable if it doesn't work after the move. That is the bug. It's is possible to set apps as none-movable.

**Comment by yashpalgoyal1304** [comment]
@chrisboyle
